app/views/tag/_miniCard.html.erb

```diff
@@ -19,9 +19,9 @@
       <% end %>
       <% if logged_in_as(['admin', 'moderator']) || (current_user && ( current_user.uid ==  @node.uid || current_user.uid == tag.uid)) %>
         <% if tag.name.include? ':' %>
-          <div class="dropdown-item"><a data-confirm="This is a power tag (see https://publiclab.org/wiki/power-tags) -- and may drive a specific function on this page. Are you sure you want to delete it?" class="tag-delete" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:black; text-decoration: underline;">delete tag</a></div>
++          <div class="dropdown-item"><a data-confirm="This is a power tag (see https://publiclab.org/wiki/power-tags) -- and may drive a specific function on this page. Are you sure you want to delete it?" class="tag-delete" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:red; text-decoration: underline;">delete tag</a></div>
         <% else %>
-          <div class="dropdown-item"><a data-confirm="Are you sure you want to delete it?" class="tag-delete" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:black; text-decoration: underline;">delete tag</a></div>
+                <div class="dropdown-item"><a data-confirm="Are you sure you want to delete it?" class="tag-delete" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:red; text-decoration: underline;">delete tag</a></div>
```

Comment:
```suggestion
          <div class="dropdown-item"><a data-confirm="Are you sure you want to delete it?" class="tag-delete" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:red; text-decoration: underline;">delete tag</a></div>
```

Reply:
Hi @giftade, feel free to just click the `Commit Suggestion` button. Thank you!!